Source/UINavigation/Private/UINavBlueprintFunctionLibrary.cpp

```diff
@@ -81,6 +83,8 @@ void UUINavBlueprintFunctionLibrary::ResetInputSettings()
 
 bool UUINavBlueprintFunctionLibrary::RespectsRestriction(FKey Key, EInputRestriction Restriction)
 {
+	FString HMD = GEngine->XRSystem->GetSystemName().ToString();
```

Comment:
Hey @goncasmage1 ,

Beginplay cannot be called in a functionlibrary since it's not an actor. We can add a getter function but for now I declared these variables at the top of the file